src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -11,7 +11,7 @@
 
   <!-- Package servicing properties -->
   <PropertyGroup>
-    <IsPackable>false</IsPackable>
+    <IsPackable>true</IsPackable>
     <VersionPrefix>4.6.0</VersionPrefix>
     <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.7.0</VersionPrefix>
```

Comment:
Why are we bumping package version from 4.6.0 to 4.7.0 @carlossanlop?

Reply:
We already published 4.6.0 on Monday https://www.nuget.org/packages/System.ValueTuple/4.6.0

So whatever we fix in this PR would need to use the next version which was merged here when we turned the packages offhttps://github.com/dotnet/maintenance-packages/pull/211/files#diff-daf2c482aa41c7a9b7f57dc5a868657fdeb25fbf8fbae292a1e3cc16f65c5666

Am I missing something?

Reply:
```suggestion
    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.1</VersionPrefix>
```

Reply:
I see what you mean. The others were bumping the minor number, this was a mistake.

https://github.com/dotnet/maintenance-packages/pull/214/files

Reply:
The 4.6.0 version was the new major bump after migrating ValueTuple to this repo. I missed that fact in my PR that turned off IsPackable and accidentally increased that same number I saw changing from the previous to the current, instead of only bumping minor. Thanks for catching it, sorry.